packages/cli/src/services/config-service.ts

```diff
@@ -26,14 +26,22 @@ export async function compileProjectAndLoadConfig(userProjectPath: string): Prom
   return readProjectConfig(userProjectPath)
 }
 
-async function compileProject(projectPath: string): Promise<void> {
+export async function compileProject(projectPath: string): Promise<void> {
   try {
     await exec('npm run clean && npm run compile', { cwd: projectPath })
```

Comment:
Now that we are adding the `boost clean` command, do we need the `npm run clean` here too? 😮

Reply:
@juanjoman I think the `npm run clean` is run before compile just in case, to avoid compilation issues. Compile was implemented that way when I started with the project. `boost clean` is just a nice to have but not necessary as you mentioned.

Reply:
Awesome, super approved then :P